tests/ttnn/unit_tests/operations/test_all_gather.py

```diff
@@ -226,11 +226,11 @@ def test_all_gather_on_t3000_post_commit_looping(
 @pytest.mark.parametrize(
     "num_devices, num_links, input_shape, dim, layout",
     [
-        (4, 2, [4, 1, 256, 32], 0, ttl.tensor.Layout.TILE),
+        # (4, 2, [4, 1, 256, 32], 0, ttl.tensor.Layout.TILE),
         (8, 1, [8, 1, 256, 32], 0, ttl.tensor.Layout.TILE),
         (8, 1, [1, 1, 32, 16384], 3, ttl.tensor.Layout.TILE),
-        (4, 2, [1, 1, 32, 32768], 3, ttl.tensor.Layout.TILE),
-        (4, 2, [4, 1, 256, 32], 0, ttl.tensor.Layout.ROW_MAJOR),
+        # (4, 2, [1, 1, 32, 32768], 3, ttl.tensor.Layout.TILE),
+        # (4, 2, [4, 1, 256, 32], 0, ttl.tensor.Layout.ROW_MAJOR),
```

Comment:
can you leave comments on whats wrong with those tests, why you had to disable?

@SeanNijjar on your radar?